iOS/MMIWG2S/ViewControllers/ViewController1.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  ViewController1.swift
+//  MMIWG2S
+//
+//  Created by Marvin Amaro on 8/23/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+import UIKit
+
+class ViewController1:UIViewController {
+    
+    var pc1View:UIView = UIView(frame: CGRect(x:0, y:0, width:0, height:0))
+    var pc2View:UIView = UIView(frame: CGRect(x:0, y:0, width:0, height:0))
+    var pc3View:UIView = UIView(frame: CGRect(x:0, y:0, width:0, height:0))
+    var pc4View:UIView = UIView(frame: CGRect(x:0, y:0, width:0, height:0))
```

Comment:
Ideally we'd want these to be contained inside of their own class so we don't have to worry about them in here. They should also be in an array.